KarhooSDKTests/TestCases/Service/Trips/KarhooTripServiceSpec.swift

```diff
@@ -71,7 +71,7 @@ final class KarhooTripServiceSpec: XCTestCase {
      *  Then:  Error should be propagated through callback
      */
     func testBookingTripRequestFailure() {
-        let tripBooking = TripBooking(quoteId: "some", passengers: mockPassengers)
+        let tripBooking = TripBooking(quoteId: "some")
```

Comment:
Might want to return this one back to the way it was.

Reply:
done